cli/ripsaw/commands/benchmark.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""CLI Commands for benchmark operations"""
+
+
+import json
+from urllib.parse import urlparse
+
+import click
+from ripsaw.clients import elastic
+from ripsaw.clients.k8s import Cluster
+from ripsaw.models.benchmark import Benchmark
+
+
+@click.group("benchmark")
+def benchmark_group():
+    """Passthrough function for Click CLI groups, used as a decorator to declare functions for the CLI"""
```

Comment:
Is this just placeholder text? (x3)

Reply:
This function leverages a feature of Click CLI to "group" commands which creates a decorator you can use (see L31). The docstring is really just there to explain why this function has to exist.

Reply:
Right, I understand that. Why don't we use that space to explain why this function has to exist to make this decorator string useful?